XLS-0073-disallow-frozen-trustline-receiving/README.md

```diff
@@ -0,0 +1,92 @@
+<pre>
+Title:       <b>XLS-73d Disallow Receiving on Frozen Trustlines</b>
+Type:        <b>draft</b>
+Revision:    <b>1</b> (2024-07-29)
+Author:      <a href="mailto:shawnxie@ripple.com">Shawn Xie</a>
+Affiliation: <a href="https://ripple.com">Ripple</a>
+</pre>
+
+#  Disallow Receiving on Frozen Trustlines
+
+## Abstract
+
+This amendment empowers token issuers on the XRP Ledger to regulate tokens to prevent misuse by blacklisted accounts. The document outlines enhancements to the interactions between frozen assets and payments, ensuring that blacklisted token holders can no longer receive funds. These changes will significantly improve use cases such as stablecoins, where ensuring regulatory compliance is essential.
```

Comment:
```suggestion
This amendment empowers token issuers on the XRP Ledger to prevent token misuse by frozen account holders. The document outlines enhancements to the interactions between frozen assets and payments, ensuring that frozen token holders cannot receive funds until or unless their trustline is unfrozen. These changes will significantly improve use cases such as stablecoins and real world assets (RWA), where ensuring regulatory compliance is essential.
```